spec/support/go_server/main.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
Don't need to change this now, but you can now alter your bridge to call the APIs I have now made available in https://github.com/temporalio/sdk-core/pull/389. So you can start a Temporalite server from code. Were it there when I built the Python SDK, I would have used it from the beginning (you'll need it later anyways).

Granted there are a couple of things Temporalite doesn't have quite yet.

Reply:
I'll have a look at it, thank you for pointing out!